### ISSUE TITLE
Browser extensions

### ISSUE THREAD

**Issue by jppferguson** [forwardemail/forwardemail.net#50]
In the Enhanced Protection plan's features, one of the features is a "Browser extension", but I can't find any other info about it... is this feature still in development?

Interested to see what the extension will be for – is it for generating new aliases?

**Comment by niftylettuce** [comment]
Yes, you can generate aliases, domains, etc all from the extension.  I should have it released soon, hopefully < 1 week.

**Comment by niftylettuce** [comment]
You will also be able to 1-click generate disposable addresses, which will auto fill in forms or copy to your clipboard (via right click menu).

**Comment by jppferguson** [comment]
Sounds awesome – looking forward to it! 🎉

**Comment by Woodham** [comment]
Hey - just wondering if this is available yet?

**Comment by niftylettuce** [comment]
Not yet - actively working on it

**Comment by Nairou** [comment]
Why is this advertised as a feature if it is still not available? It gives a false impression when comparing to other companies who do already have browser extensions.

**Comment by Ionshard** [comment]
Is this still not available? This issue has been open for well over a year. When it's due to be released "soon" and it's on the pricing page that's a little sketchy but possibly forgivable. If it's been on the pricing page for this long without being available ... that's getting into scam territory.

**Comment by hagai-helman** [comment]
For both ethical and legal reasons, it should be removed from the pricing page immediately, until the extension is released.

**Comment by niftylettuce** [comment]
I'm adding a "(coming soon)" and will ping you once this is out.

**Comment by niftylettuce** [comment]
<img width="464" alt="Screen Shot 2021-11-11 at 6 55 56 PM" src="https://user-images.githubusercontent.com/523141/141385520-e38c0da1-4fca-4b0f-a014-2bad854fdd2a.png">

**Comment by shaunwarman** [comment]
@Douglas-Cox any takeaways from that browser extension you liked most and anything on the wish list?
